snapshots/manager.go

```diff
@@ -314,6 +314,11 @@ func (m *Manager) shouldTakeSnapshot(height int64) bool {
 func (m *Manager) snapshot(height int64) {
 	m.logger.Info("creating state snapshot", "height", height)
 
+	if height < 0 {
```

Comment:
should this be <= 0 (I asked based on the error message)?

Reply:
Yes, you are right. I will get that changed